assets/src/public.scss

```diff
@@ -1195,6 +1195,7 @@ body {
     &.attendance-inactive  div.attendance-option {
         color: $color_alert;
     }
+    &.attendance-online.address-specific p.location-address,
```

Comment:
we have a part commented out after this that says
```
    &.tsml-meeting.tsml-type-tc {
        p.location-address,
        address {
            color: #999;
            text-decoration: line-through;
        }
    }
```
wouldn't it have been easier to use that than to add a new `address-specific` class?